GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -1282,10 +1282,10 @@ private void selectTrackedToolStripMenuItem_Click(object sender, EventArgs e)
             SetBranchesPushCheckboxesState(row =>
                 {
                     // Check if the branch is tracked (i.e. not new)
-                    var localColumn = row.Cells[LocalColumn.Name] as DataGridViewTextBoxCell;
-                    var remoteColumn = row.Cells[RemoteColumn.Name] as DataGridViewTextBoxCell;
-                    return localColumn is not null && remoteColumn is not null
-                           && !string.IsNullOrEmpty(localColumn.Value.ToString()) && !string.IsNullOrEmpty(remoteColumn.Value.ToString());
+                    return row.Cells[LocalColumn.Name] is DataGridViewTextBoxCell localColumn &&
+                           row.Cells[RemoteColumn.Name] is DataGridViewTextBoxCell remoteColumn &&
+                           !string.IsNullOrEmpty(localColumn.Value.ToString()) &&
+                           !string.IsNullOrEmpty(remoteColumn.Value.ToString());
```

Comment:
Left Reading works better with operators at the begin of the next lines.

Reply:
Is there an analyzer we could use to control this uniformly across the code?

Reply:
I'm not aware of any. 
@sharwell is the Master of Analyzers, maybe he knows :)